### ISSUE TITLE
Label size limit

### ISSUE THREAD

**Issue by cristovao-trevisan** [plasmatech8/vscode-better-sveltekit-tab-labels#1]
I need this extension so bad :smiling_face_with_tear: 
I thought of building it and submit it to the svelte hackathon, but found your work and didn't lose much time to realize it's impossible for now.

One quick suggestiion for when it becomes possible:
A way to limit the size of the tab name. May be number or chars but I think a max of folders would be better.
Eg. with `maxFolders: 2`
> ` /src/routes/my/deep/nested/route/for/user/auth` -> `.../user/auth`

**Comment by plasmatech8** [comment]
Yeah, this extension is not possible for now. Unless there is some fancy way to get around the problem.

Currently only way to _prettify_ your tabs is to update `workbench.editor.labelFormat` in the settings. (though I am not fond of anything other than default)

I must have forgotten to add some notes + links for reference, so thanks for the reminder. 🙂
> https://github.com/microsoft/vscode/issues/41909
> https://code.visualstudio.com/api/references/vscode-api#Tab

**Comment by plasmatech8** [comment]
@cristovao-trevisan looks like VSCode maintainers heard our prayers (see #2)

I don't know if there is much level of customizability, or exposure to VSCode extensions, but looks good to me.
